Add missing MediaList subpages (#24386)
mdn/content: files/en-us/web/api/medialist/item/index.md

---
title: MediaList.item()
slug: Web/API/MediaList/item
page-type: web-api-instance-method
browser-compat: api.MediaList.item
---

{{ APIRef("CSSOM") }}

The **`item()`** method of the {{domxref("MediaList")}} interface returns the media query at the specified `index`, or `null` if the specified `index` doesn't exist.

## Syntax

```js-nolint
item(index)
[index]
```

> **Note:** The natural bracket (`[]`) syntax can be used instead of the more verbose `item()` syntax.

### Parameters

- `index`
  - : An integer.

### Return value

> A string, or `null` if there is no entry for the index.
> 
> If there is no entry for the given index, and using the bracket (`[]`) syntax, `undefined` is returned.

Reviewer: ```suggestion
If the bracket (`[]`) syntax is used and there is no entry for the given index, `undefined` is returned.
```

(just a suggestion that IMO reads a little better)
